lib/oca-epak/oep/client.rb

```diff
@@ -17,10 +17,12 @@ def initialize(username, password)
       # @return [String] HTML
       def get_html_de_etiquetas_por_orden_or_numero_envio(opts = {})
         method = :get_html_de_etiquetas_por_orden_or_numero_envio
-        opts = { "idOrdenRetiro" => opts[:id_orden_retiro],
-                 "nroEnvio" => opts[:nro_envio] }
+        opts = {
+          "idOrdenRetiro" => opts[:id_orden_retiro],
```

Comment:
@cavi21 hm, shouldn't this be "IdOrdenRetiro"?

Reply:
I know this is _slightly_ off topic, but, where do you get the `idOrdenRetiro` value? It doesn't seem to match the `codigooperacion` returned when creating a pickup order.

Reply:
hi @WhyNotHugo! it's being a while from the last time I used this gem/service. That's why I don't have this information on the top of my head. 
Tonight I'll try to check about this `idOrdenRetiro` in order to give you more insight about it.

Also I owe a bunch of things on this PR to be able to be merge 😞. Next week I'll try to work on them.

Reply:
Thanks for looking into it, look forward to hearing back! ^^